repo/packages/R/redis/0/marathon.json.mustache

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "{{redis.framework-name}}",
+    "container": {
+        "docker": {
+            "image": "{{resource.assets.container.docker.redis-docker}}",
+            "network": "HOST",
+            "forcePullImage": true
+        },
+        "type": "DOCKER",
+        "volumes": []
+    },
+    "args": [],
+    "cpus": {{redis.cpus}},
+    "mem": {{redis.mem}},
+    "instances": 1,
+    "ports": [5454, 8080, 2379],
+    "env": {
+      "FRAMEWORK_NAME":"{{redis.framework-name}}",
+      "CLUSTER_SIZE":"{{redis.cluster-size}}",
+      "MESOS_MASTER":"{{redis.master}}"
+    },
+    "healthChecks": [{
+      "protocol": "COMMAND",
+      "command": { "value": "/usr/bin/curl -f -k https://$HOST:2379/healthz" },
```

Comment:
This is not going to be expanded. You probably want to use the builtin HTTP health checks.